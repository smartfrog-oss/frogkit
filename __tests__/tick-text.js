import { mount } from 'vue-test-utils'
import TickText from '@/components/tick-text'

describe('Tick Text component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(TickText, {propsData: {placeholder: 'some text goes here', value: 'another text' }})
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('should render component with bold text', () => {
    const wrapper = mount(TickText, {propsData: {placeholder: 'some text goes here', value: 'another text', bold: true }})
    expect(wrapper.html()).toMatchSnapshot()
  })

})
