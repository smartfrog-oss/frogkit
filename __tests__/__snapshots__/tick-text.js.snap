// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tick Text component should render component and match snapshot 1`] = `
<section class="fk-tick-text fk-tick-text--orange">
    <flex align="top" class="fk-tick-text__container">
        <icon icon="checkmark" color="orange" class="fk-tick-text__icon"></icon>
        <flex column="" grow="">
            <div class="fk-tick-text__placeholder">some text goes here</div>
        </flex>
    </flex>
</section>
`;

exports[`Tick Text component should render component with bold text 1`] = `
<section class="fk-tick-text fk-tick-text--orange fk-tick-text--bold">
    <flex align="top" class="fk-tick-text__container">
        <icon icon="checkmark" color="orange" class="fk-tick-text__icon"></icon>
        <flex column="" grow="">
            <div class="fk-tick-text__placeholder--bold">some text goes here</div>
        </flex>
    </flex>
</section>
`;
